Introduction/Introduction.md

```diff
@@ -0,0 +1,23 @@
+Kernel introduction.
+================================================================================
+
+The Begining
+--------------------------------------------------------------------------------
+Linux has began in the Operating system more important in the last years and many people use it in their lives commonly.
```

Comment:
Not quite sure what you're trying to say here. Perhaps: Linux' importance as an operating system has grown in recent years, and many people use it in their daily lives.